docs/examples/notebooks/04-oscillator.ipynb

```diff
@@ -247,16 +251,65 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "fig, ax, anim = make_anim_simple_osc(sim.data[output_p][:, 0],\n",
-    "                                     sim.data[output_p][:, 1])\n",
-    "HTML(anim.to_html5_video())"
+    "def make_anim():\n",
+    "    _, _, anim = make_anim_simple_osc(sim.data[output_p][:, 0],\n",
+    "                                      sim.data[output_p][:, 1])\n",
+    "    return anim\n",
+    "\n",
+    "\n",
+    "HTML(make_anim().to_html5_video())"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Try playing around with the number of neurons in the FPGA ensemble as well as the synaptic time constant (`tau`) to see how it effects performance (e.g., observe how changing these numbers affect the stability of the oscillator)! Additionally, modify the oscillation frequency (try making it negative) to its impact on the output. You can also change the simulation time and see how long the oscillator is stable for!"
+    "## Step 9: Spiking Neurons\n",
+    "\n",
+    "The plots above demonstrate the results of a simple oscillator implemented in a network of non-spiking linear rectified neurons. The network can also be simulated using spiking neurons to illustrate the similarities and differences between a spiking and a non-spiking network.\n",
```

Comment:
Same comment as other notebook, rectified linear vs linear rectified. also we should pick rate or non-spiking everywhere.